<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VANTSO Kalite ve Akreditasyon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Müşteri Memnuniyet Yönetimi Sistemi</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>437</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ISO 10002
                                Müşteri Memnuniyet Yönetim Sistemi</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Kalite Yönetimi Sistemi</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1011</width>
        <height>437</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ISO 9001 : 2015
                                Kalite Yönetimi Sistemi</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Performans Değerlendirme</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>441</height>
       </rect>
      </property>
      <column>
                 <property name="text">
        <string>PD</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Çalışan Memnuniyeti</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>441</height>
       </rect>
      </property>
      <column>
            <property name="text">
        <string>ÇM</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Belge Görüntüle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Belge Ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>570</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Belge Sil</string>
    </property>
   </widget>
   <widget class="QLabel" name="pdfLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>1001</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
